config/locales/en.yml

```diff
@@ -104,3 +104,102 @@ en:
+      github_url: ''
+    - name: Bill Walker
+      image: hall_of_fame_images/bill.jpg
+      github_url: http://github.com/mach1010
+    - name: Mazin Power
+      image: https://avatars1.githubusercontent.com/u/7075351?v=3&s=400
+      github_url: https://github.com/muzfuz
+    - name: Ada Chiu
+      image: https://avatars0.githubusercontent.com/u/13413986?v=3&s=400
+      github_url: https://github.com/adachiu
+    - name: Kevin Mulhern
+      image: https://avatars1.githubusercontent.com/u/7963776?v=3&s=460
+      github_url: https://github.com/KevinMulhern
+    - name: Arun Kumar
+      image: https://avatars0.githubusercontent.com/u/7039523?v=3&s=400
+      github_url: https://github.com/arku
+    - name: Cody Loyd
+      image: https://avatars0.githubusercontent.com/u/18574792?v=3&s=400
+      github_url: https://github.com/codyloyd
+    - name: Cornelius Soon
+      image: https://avatars2.githubusercontent.com/u/16495574?v=3&s=400
+      github_url: https://github.com/csrail
+    - name: Rhys
+      image: https://avatars3.githubusercontent.com/u/19835799?v=3&s=400
+      github_url: https://github.com/105ron
+    - name: Austin
+      image: https://avatars2.githubusercontent.com/u/12649577?v=3&s=400
+      github_url: https://github.com/CouchofTomato
+    - name: Leo
+      image: https://avatars2.githubusercontent.com/u/13614862?v=3&s=400
+      github_url: https://github.com/leosoaivan
+    - name: Chad Kreutzer
+      image: https://avatars1.githubusercontent.com/u/13308917?v=3&s=400
+      github_url: https://github.com/ChadKreutzer
+    - name: Roberto Pando
+      image: https://avatars1.githubusercontent.com/u/20132181?v=3&s=400
+      github_url: https://github.com/RobPando
+    - name: Ryan
+      image: https://avatars3.githubusercontent.com/u/20901017?v=3&s=400
+      github_url: https://github.com/ryanford-frontend
+  ways_to_contribute:
+    - title: Contribute to the Curriculum
+      description: |
+        The curriculum consists of the lessons and projects on this site.
+        We are constantly expanding and updating the curriculum to keep it current and
+        ensure it meets our students' needs in the ever changing world of web development.
+        Our goal is to create the best free web development curriculum in the world, anyone
+        is welcome to join in and help us in striving for this goal.
+      path: curriculum
+    - title: Contribute to the Main Site
+      description: |
+        The site you are on right now is a Ruby on Rails application. We are
+        always working on new features that improve the platform and help students in
+        their learning journey. As with any Open Source project, there is also a constant
+        backlog of outstanding issues and bugs that need to be fixed. Anyone is welcome
+        to join in and start working on any of the issues we currently have posted in
+        the Github repository for the site.
+      path: theodinproject
```

Comment:
@KevinMulhern Now that the the translation file is seeing a bit of use, thoughts on refactoring to use proper translation key structure throughout? ie 

`en.static_pages.contributing.contributing_options.ways_to_contribute`

for this particular case / etc. 

A little bit of extra overhead but lays a good pattern for future use. Can do this in a separate ticket if we're keen

Reply:
💯 good idea, the file is getting a bit unwieldy aha